app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -1,10 +1,26 @@
 ManageIQ.angular.app.controller('cloudVolumeFormController', ['$http', '$scope', 'cloudVolumeFormId', 'miqService', function($http, $scope, cloudVolumeFormId, miqService) {
-  $scope.cloudVolumeModel = { name: '', cloud_tenant_id: '' };
+  $scope.cloudVolumeModel = { 
+    name: '', 
+    storage_manager_id: '', 
+    cloud_tenant_id: '',
+    availability_zone_id: '',
+    aws_iops: '',
+    aws_volume_type: ''
+  };
   $scope.formId = cloudVolumeFormId;
   $scope.afterGet = false;
   $scope.modelCopy = angular.copy( $scope.cloudVolumeModel );
   $scope.model = "cloudVolumeModel";
 
+  // This ia a fixed list of available cloud volume types for AWS.
+  $scope.awsVolumeTypes = [
+    { type: "gp2", name: "General Purpose SSD (GP2)" },
+    { type: "io1", name: "Provisioned IOPS SSD (IO1)" },
+    { type: "st1", name: "Throughput Optimized HDD (ST1)" },
+    { type: "sc1", name: "Cold HDD (SC1)" },
+    { type: "magnetic", name: "Magnetic" }
+  ];
+
```

Comment:
Yes, it's fixed. The question is whether this is something the provider should offer. Something like the list of regions [here](https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/app/models/manageiq/providers/amazon/regions.rb).